Themes/default/Display.template.php

```diff
@@ -688,9 +687,8 @@ function template_single_post($message)
 								</div>';
 
 	echo '
-								<span class="page_number floatright">
-									', !empty($message['counter']) ? ' #' . $message['counter'] : '', ' ', '
-								</span>
+								
```

Comment:
Minor issue: Please remove the empty line here.

Reply:
Oops